docs/_panels/attracted-contributors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Attracted Contributors
+description: attraction of contributors for Git and GitHub.
+author: Bitergia
+screenshot: sigils/attracted-contributors.png
+created_at: 2023-01-09
+grimoirelab_version: 0.2.2
+layout: panel
+---
+
+This panel shows the evolution of attracted contributors in the last ten years and the
+ones who recently joined. This analysis uses each person's first contribution date to
+estimate when they enter the project.
+
+This panel is intended to be used with the whole history of the community activated and
+not with some pre-established activity timeframe as in other panels (for instance,
+2 years). This is done this way as the information pre-calculated and displayed focuses
+on the first contribution of each new member.
+
+The drop-down menu on the top left side allows you to switch from Git to GitHub data sets,
```

Comment:
I don't know if you have established a filter to limit the options to `git` and `github`, but in other case, there could be more options to choose if there is another data source with demography data, such as Gerrit.

Reply:
I'm using the same viz used the `Demographics` dashboard uses. As far as I saw it is not limited to `git` and `github` and it should not be. There is also support for `gerrit`!